src/services/codefixes/importFixes.ts

```diff
@@ -24,16 +24,14 @@ namespace ts.codefix {
         moduleSpecifier?: string;
     }
 
-    interface SymbolContext extends textChanges.TextChangesContext {
+    interface SymbolContext {
         sourceFile: SourceFile;
         symbolName: string;
+        formatContext: ts.formatting.FormatContext;

```

Comment:
Looks like this could be moved to `ImportCodeFixContext`.

Reply:
It looks like every consumer of `SymbolContext` ands it with `TextChangesContext` anyway, so I'll just make it a subtype.

Reply:
Do you know why `SymbolContext` exists?  Is it to prevent certain functions from accessing certain members of `ImportCodeFixContext`?

Reply:
It's just there to indicate that we don't need everything from `ImportCodeFixContext`.

Reply:
That seems excessive to me, but not worth changing at the moment.